src/julia/darray.jl

```diff
@@ -128,6 +161,21 @@ function foxLi{T<:BlasComplex}(::Type{T}, n::Integer, ω::Real)
 end
 foxLi(n::Integer, ω::Real) = foxLi(Complex128, n, ω)
 
+function uniform{T<:BlasFloat}(::Type{T}, m::Integer, n::Integer)
```

Comment:
I am essentially doing the exact same thing (see `src/core/global.cpp`), so I am A-okay with using `drand` instead. Would you mind submitting the revision?